### PR TITLE
Beautification and Resources

### PR DESCRIPTION
**Pull request by BekaValentine**
This PR beautifies the page, and adds resources that folx might find helpful when going to a protest.

### CODE REVIEW

File: findmyprotest.html [hunk 1]
```diff
@@ -1,89 +1,145 @@
...
+			<h2>Resources</h2>
+			<p>
+				<a href="https://www.nlg.org/chapters/">National Lawyers Guild</a>
+				(Find your local NLG number and memorize it, write it on your arm in permanent marker, etc.)
+			</p>
+			<p>
+				<a href="https://en.wikipedia.org/wiki/Tactics_and_methods_surrounding_the_2019%E2%80%9320_Hong_Kong_protests">Tactics and methods surrounding the 2019–20 Hong Kong protests</a>
+			</p>
+			<p>
+				<a href="https://www.bloomberg.com/graphics/2019-hong-kong-protesters-umbrellas/">The Essential Tool for Hong Kong Protesters? An Umbrella</a>
```
Comment (eiais) [inline findmyprotest.html]: got a paywall on this, maybe switch to an archive link?

File: findmyprotest.html [hunk 2]
```diff
@@ -1,89 +1,145 @@
...
-		<div class="ol"><input type="checkbox" id="nearbyarea"><label for="nearbyarea">Show nearby areas (not implemented)</label></div>
-		<div class="ol"><input type="checkbox" id="nearyou"><label for="nearyou">Limit to tweeters near you</label></div>
-		<div class="ol"><input type="checkbox" id="BLMT"><label for="vehicle1">Black Lives Matter terms</label></div>
-		<div class="ol"><input type="checkbox" id="PT"><label for="vehicle1">Protest terms</label></div>
-		<div class="ol"><input type="checkbox" id="RT"><label for="vehicle1">Riot terms</label></div>
-		<div class="ol"><input type="checkbox" id="LEOT"><label for="vehicle1">Law Enforcement terms</label></div>
-		<div class="ol"><button onclick="directToTwitter()">Find My Protest</button></div>
-		<a href="https://forms.gle/1dgCaQmyeRqUQ36R9">Send Feedback</a>
+		<div>
+			<p>This will search twitter with the types of keywords you check to help you find the protests in your city</p>
```
Comment (eiais) [inline findmyprotest.html:83]: Can you add a header to this to match the other two?

File: findmyprotest.html (hunk 1 above)
Comment (BekaValentine) [reply findmyprotest.html]: Done!

File: findmyprotest.html (hunk 2 above)
Comment (BekaValentine) [reply findmyprotest.html:83]: Done!

**Review by eiais — APPROVED** [review]

**Pull request merged**